lib/stdlib/doc/src/math.xml

```diff
@@ -102,9 +102,20 @@ erf(X) = 2/sqrt(pi)*integral from 0 to X of exp(-t*t) dt.</pre>
 
     <func>
       <name name="pi" arity="0" since=""/>
-      <fsummary>A useful number.</fsummary>
+      <fsummary>Ratio between circumfence and diameter of the circle.</fsummary>
```

Comment:
```suggestion
      <fsummary>Ratio of circumfence to diameter of a circle.</fsummary>
```
(actually, to be _very_ pedantic, the ratio of circumference to diameter of a circle ***on a plane*** :grin: (just joking))